algorithms/ar-bsrt/csharp/BubbleSort.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace BubbleSort
+{
+    public class BubbleSort

```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)